portal/models/user.py

```diff
@@ -1575,8 +1575,8 @@ def delete_user(self, acting_user):
 
         if self == acting_user:
             raise ValueError("can't delete self")
-        if self is None or acting_user is None:
-            raise ValueError("both user and acting_user must be well defined")
+        if acting_user is None:
```

Comment:
Just curious, why not `not acting_user` vs `is None`?

Reply:
no good reason - is None is clearly too specific for this case.  will fix